ichnaea/locate/searcher.py

```diff
@@ -118,6 +120,9 @@ class PositionSearcher(Searcher):
         ('wifi', (
             WifiPositionProvider,
         )),
+        ('fallback', (
+            FallbackProvider,
+        )),
```

Comment:
Could we reorder this list and put the geoip provider last? Would that avoid the "weak location" check inside the fallback's should_locate method?